nodes/srl/srl.go

```diff
@@ -586,6 +592,16 @@ func (n *srl) addDefaultConfig(ctx context.Context) error {
 	// prepare the endpoints
 	for _, e := range n.Endpoints {
 		ifName := e.GetIfaceName()
+		if ifName == "mgmt0" {
+			// skip the mgmt0 interface. This is just for traffic carrying interface
+
+			// if the endpoint has a custom MTU set, use it in the template logic
+			// otherwise we don't set the mtu as srlinux will use the default max value 9232
+			if m := e.GetLink().GetMTU(); m != links.DefaultLinkMTU {
+				tplData.MgmtMTU = m
+			}
```

Comment:
I think also another condition can be copied from the generic mtu handling where we do not allow setting mtu less than 1500

Reply:
not needed, the interface won't come up if the mtu is lower than 1500 anyways, so the check is not quite useful